### PR TITLE
[vb/en] typos correction. type inconsistencies fix. Improvements

### PR DESCRIPTION
**Pull request by ioab**

### CODE REVIEW

File: visualbasic.html.markdown [hunk 1]
```diff
@@ -126,10 +126,10 @@ Module Module1
         'Of course we would like to be able to add up decimals.
         'Therefore we could change the above from Integer to Double.
 
-        'Enter a whole number, 1.2, 2.4, 50.1, 104.9 ect
+        'Enter a floating-point number, 1.2, 2.4, 50.1, 104.9 etc
```
Comment (vendethiel) [inline visualbasic.html.markdown]: please add a comma for consistency with L113 :)

**Comment by ioab** [comment]
okay.

**Comment by vendethiel** [comment]
Thanks! :cake:

**Comment by ioab** [comment]
Thank you :smile:

**Pull request merged**